src/lib.rs

```diff
@@ -16,19 +16,18 @@
 //!
 //! This crate also provides the following attributes:
 //!
-//! - [`#[entry]`] to declare the entry point of the program
-//! - [`#[exception]`] to override an exception handler. If not overridden all exception handlers
+//! - `#[entry]` to declare the entry point of the program
+//! - `#[exception]` to override an exception handler. If not overridden all exception handlers
 //!   default to an infinite loop.
-//! - [`#[pre_init]`] to run code *before* `static` variables are initialized
-//!
-//! [`#[entry]`]: ../cortex_m_rt_macros/fn.entry.html
-//! [`#[exception]`]: ../cortex_m_rt_macros/fn.exception.html
-//! [`#[pre_init]`]: ../cortex_m_rt_macros/fn.pre_init.html
+//!   default to an infinite loop.
```

Comment:
You have this line twice for some reason.

Reply:
removed